### PR TITLE
updated changelogs for remaining releases

### PR DESCRIPTION
**Pull request by jfmoron**
Updated the changelings for the following releases

Mac

- 1.1.0
- 1.1.1
- 1.1.2
- 1.1.3
- 1.1.4

Windows

- 1.1.1
- 1.1.2
- 1.1.3

### CODE REVIEW

File: mac/CHANGELOG.md [hunk 1]
```diff
@@ -5,10 +5,79 @@ Kandy Distant Driver for VDI Mac change log.
 - This change log follows [keepachangelog.com](http://keepachangelog.com/) recommendations.
 - This project adheres to [Semantic Versioning](http://semver.org/).
 
+# 1.1.4 - 2021-12-07
```
Comment (brenthompson) [inline mac/CHANGELOG.md:8]: not a big deal, but i noticed that the Mac changelog uses # for versions, the one for Windows uses two ##. Afaik it should be 2, but it probably doesn't matter, but it should be consistent.

**Review by brenthompson — CHANGES_REQUESTED** [review]
one minor thing to cleanup i think

File: mac/CHANGELOG.md (hunk 1 above)
Comment (jfmoron) [reply mac/CHANGELOG.md]: fixed

**Review by brenthompson — APPROVED** [review]
i would have put the Known Issues either in the respective readme's, or under the Release they apply to. Not sure they fit in with the statement "This change log follows [keepachangelog.com](http://keepachangelog.com/) recommendations." since it doesn't mention Known Issues or limitations. They do have a release type called Unreleased to show what's coming later, which we could use instead of Known Issues.
but, up to you. i dont think it's worth spending a lot of time on documentation no one will ever read. we can change it for future releases

**Pull request merged**